src/components/search.js

```diff
@@ -225,12 +230,19 @@ function Search(props) {
           onClick={() => {
             setSearchValue('');
             setResults([]);
+            setSearchDone(false);
           }}
         >
           <Icon.X />
         </div>
       )}
 
+      {searchDone && results.length === 0 && (
+        <div className="results">
+          <h3 style={{textAlign: 'center'}}>No Results Found</h3>
```

Comment:
Can you add a pop up for the message of No results Found?
Or you can add a screenshot for more clarity.